src/cart/crt_iv.c

```diff
@@ -1070,21 +1099,31 @@ handle_ivfetch_response(const struct crt_cb_info *cb_info)
 	ivns = iv_info->ifc_ivns_internal;
 	class_id = iv_info->ifc_class_id;
 
+	/* Check to see if version change on while waiting for reponse */
+	/* Not an error. But good to know.			     */
+	local_version = ivns->cii_grp_priv->gp_membs_ver;
+	if (local_version != rpc_version) {
+		D_DEBUG(DB_ALL, "Group Version Changed: ");
+		D_DEBUG(DB_ALL, "rpc_version 0x%x: Local version 0x%x\n",
+				rpc_version, local_version);
```

Comment:
multiple debug prints like this are better combined into a single one, otherwise during high load/transaction situation with multiple entities logging into same file those statements can be printed far apart from each other making it harder to debug

Reply:
I had split this to stop checkpatch from throwing out warnings.  I will recombined this (and all other situations that I split)

Reply:
Split comments are combined.